### ISSUE TITLE
Appium Desktop cannot navigate to all elements in the App Source

### ISSUE THREAD

**Issue by rdhandapani88** [appium/appium-desktop#1347]
## Appium or Appium Desktop?

You are reporting an issue at the Appium Desktop repository. Appium Desktop is a wrapper around Appium. If you are having trouble running tests, it is much more likely that the problem you are encountering is not a problem with Appium Desktop but with Appium. For that reason we require you to have tried your testcase with Appium CLI before reporting issues. Check the checkbox below to confirm that you have proven your issue does _not_ reproduce on Appium itself:

- [ Yes] I have verified that my issue does not occur with Appium and should be investigated as an Appium Desktop issue

## The problem

Appium Desktop 1.17.0, cannot swipe to all elements in app source. The window cannot dragged till bottom to find all elements. 
## Environment

1.17.0

- I am running Appium Desktop version 1.17.0
- I am on (pick one):
    - [Yes ] Mac

**Comment by KazuCocoa** [comment]
Do you mean https://github.com/appium/appium-desktop/pull/1327 ?

**Comment by rdhandapani88** [comment]
Yes , exactly same issue still persist. we cannot scroll vertical.

**Comment by KazuCocoa** [comment]
Okay, thanks.
The fix is available on master branch for now
